lib/Interpreter/CIFactory.cpp

```diff
@@ -316,14 +316,9 @@ namespace {
       }
 
     #if defined(__GLIBCXX__)
-      // Avoid '__float128 is not supported on this target' errors
```

Comment:
Uh okay, that was easy. It's not an arch issue. type_traits:344 has
```
    342 #if !defined(__STRICT_ANSI__) && defined(_GLIBCXX_USE_FLOAT128)
    343   template<>
    344     struct __is_floating_point_helper<__float128>
```
I.e. we don't have `__STRICT_ANSI__` defined for 1z. Okay, now I see the relevance of -std=c++11 - this defines `__STRICT_ANSI__`. (So this is about `__STRICT_ANSI__` and `__float128` is just a symptom.)

I need to think... But either way, this comment is out of context.